controllers/noteController.js

```diff
@@ -0,0 +1,92 @@
+const noteRepository = require("../repositories/noteRepository");
```

Comment:
You can rename this file to note instead as the folder name indicates that this is a controller.